### DISCUSSION TITLE
Is there a way to rebuild metadata locally?

### DISCUSSION THREAD

**Discussion by ilyabezdelev** [avitorio/outstatic#247] [Q&A]
My workflow is such that I create a post in Outstic, then edit it locally in markdown, then publish with a PR.

After composing the post, I often change the slug, the title, and the description. I'm finding myself changing the metadata both in the markdown file and in `metadata.json`.

Is there a way to rebuild the metadata file locally, so it could be a part of the same PR as the content?

Thanks!

**Comment by avitorio** [comment]
Hey @ilyabezdelev,

As I've mentioned in my previous post, Outstatic doesn't support local editing.
You can set your OST_REPO_BRANCH locally to a different branch if you don't want to commit to main. Then create the PR from that branch.

I hope this works.
